UACloudLibraryServer/Startup.cs

```diff
@@ -247,6 +284,37 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddServerSideBlazor();
+
+            // Required to make Azure AD login work as ASP.Net External Identity
+            // From https://stackoverflow.com/a/71360559:
+            // Change the SignInScheme to External after ALL other configuration have run.
+            services
+              .AddOptions()
+              .PostConfigureAll<OpenIdConnectOptions>(o => {
+                  o.SignInScheme = IdentityConstants.ExternalScheme;
+              });
+        }
+
+        private static Task CheckRedirectForApi(RedirectContext<CookieAuthenticationOptions> ctx)
+        {
```

Comment:
Can move this into the basic auth handler, which should always fire? Distributing something important like auth around a project is a bug farm. :-)

Reply:
Figure out a way to make this work. Removed all these checks.